cli/commands/rewards/fetchevents.ts

```diff
@@ -54,10 +54,19 @@ export default class FetchEvents extends BaseCommand {
     }
 
     const progressBar = cli.progress()
-    const progressBarTotal = Math.floor(((toBlock - fromBlock)/batchSize + 1) * 2)
+    const progressBarTotal = Math.floor(((toBlock - fromBlock)/batchSize + 1) * 3)
```

Comment:
What's this magic number?

Reply:
this ultimately equals the number of times I go fetch events from forno. Which is how many batches I need for the span of blocks times 3 which is the 3 distinct event types I query. I didn't go for intense readability here since it's only for a progressbar UX number, but am open to making this less "magic" lol